packages/office-ui-fabric-react/src/utilities/decorators/withResponsiveMode.tsx

```diff
@@ -59,48 +59,42 @@ export function withResponsiveMode<TProps extends { responsiveMode?: ResponsiveM
 ): any {
   const resultClass = class WithResponsiveMode extends BaseDecorator<TProps, IWithResponsiveModeState> {
     private _events: EventGroup;
+    private _async: Async;
 
     constructor(props: TProps) {
       super(props);
       this._events = new EventGroup(this);
+      this._async = new Async(this);
       this._updateComposedComponentRef = this._updateComposedComponentRef.bind(this);
 
-      this.state = {
-        responsiveMode: getInitialResponsiveMode(),
```

Comment:
Will this change regress the performance optimization made for SharePoint Online in https://github.com/microsoft/fluentui/pull/12258?

Reply:
what is the initial state now? we had an issue where the default initial mode was not correct, and it quickly triggers a re-render after detecting the new window size. Kevin added the getIntialResponsiveMode to fix this issue before.